examples/with-segment-analytics/lib/segment.ts

```diff
@@ -0,0 +1,5 @@
+import { AnalyticsBrowser } from '@segment/analytics-next'
+
+export const analytics = AnalyticsBrowser.load({
```

Comment:
Are you sure this works? App directory components are server components by default. Calling `.load()` as a module export feels like it might cause errors for things like `window` being unavailable. I believe this is why the official Segment example places initialization within a context provider.

Reply:
Oh, I see. The only usage you have is within client components.